src/collector/rrdp/http.rs

```diff
@@ -256,6 +259,29 @@ impl HttpClient {
         }
     }
     */
+
+    /// The redirect policy.
+    ///
+    /// We allow up to 10 redirects (reqwest’s default policy) but only if
+    /// the origin stays the same.
+    fn redirect_policy(attempt: redirect::Attempt) -> redirect::Action {
+        if attempt.previous().len() > 9 {
+            return attempt.stop();
+        }
+        let orig = match attempt.previous().first() {
+            Some(url) => url,
+            None => return attempt.follow() // Shouldn’t happen?
```

Comment:
Is there no more conservative alternative instead of `.follow()`? If this is really unexpected, perhaps a warning should be logged. Perhaps combined with a `return attempt.stop()`?
A warning might actually be quite useful for (repo) operators to troubleshoot presumably unexpected behavior.

Reply:
If the alternative is to (silently) stop processing this RRDP repo, maybe this is better indeed.